docs/conf.py

```diff
@@ -9,7 +9,9 @@
 
 # django-specific configuration
 from django.conf import settings
-settings.configure()
+
+# Removed for breaking doc building with django-guardian installed.
+# settings.configure()
```

Comment:
Additional note:

This issue is why [these lines](https://github.com/mitodl/lore/pull/117/files#r31921279) (the environment variables) were added to `tox.ini`.